integration/feature/init/src/MillInitMavenTests.scala

```diff
@@ -0,0 +1,221 @@
+package mill.integration
+
+import mill.testkit.{IntegrationTester, UtestIntegrationTestSuite}
+import utest._
+
+object MillInitMavenTests extends UtestIntegrationTestSuite {
+
+  private def prep[T](githubSourceZipUrl: String)(f: IntegrationTester => T): T = {
+    val zipFile = os.temp(requests.get(githubSourceZipUrl))
+    val unzipDir = os.unzip(zipFile, os.temp.dir())
+    val sourceDir = os.list(unzipDir).head
+
+    integrationTest { tester =>
+      for (p <- os.list(sourceDir)) os.move.into(p, tester.workspacePath)
+      f(tester)
+    }
+  }
+
+  def tests: Tests = Tests {
+
+    // - Junit5
+    // - maven-compiler-plugin release option
+    test("jansi") {
+      prep("https://github.com/fusesource/jansi/archive/refs/tags/jansi-2.4.1.zip") { tester =>
+        import tester._
+
+        val initRes = eval(("init", "--deps-object", "Deps"))
+        initRes.out.contains(
```

Comment:
I think these checks will all be easier to read converted to normal `assert`s. `==>` is generally meant for user-facing test cases to make them prettier, but these are just internal so `assert` is fine